API.md

```diff
@@ -445,4 +376,40 @@ Returns:
 Error responses
 
 - `400` - missing mandatory parameter
-- `404` - invitation not found
\ No newline at end of file
+- `404` - invitation not found
+
+## Render
+
+### POST /render
+
+Parameters:
+
+- `choirId` - the id of choir whose song is being rendered (required)
+- `songId` - the id of the song being rendered (required) 
+- `status` - one of `new`/`converted`/`aligned`/`rendered` (default `new`)
+
+Returns:
+
+```js
+{
+  ok: true
+}
+```
+
+### GET /render
+
+Parameters:
+
+- `choirId` - the id of the choir whose song is being rendered (required)
+- `songId` - the id of the song being rendered (required)
+
+Returns:
```

Comment:
Will this return only a single record, or will it return all of the records for the different states  `(new/converted/aligned/rendered)` etc?